### PR TITLE
Only show get started when available

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8396/

**Comment by makyo** [comment]
[Move get started link](https://trello.com/c/fASkWWrq/176-move-get-started-link)

File: jujugui/static/gui/src/app/components/inspector/header/header.js [hunk 1]
```diff
@@ -75,21 +75,21 @@ class InspectorHeader extends React.Component {
     if (!this.props.showLinks) {
       return;
     }
-    const classList = classNames(
-      'inspector-header__list-item',
-      {'inspector-header__list-item--disabled': !this.props.hasGetStarted});
+    const getStarted = !this.props.hasGetStarted ? '' : (
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/header/header.js]: I usually prefer to do a 'positive' conditional, but whichever :)

File: jujugui/static/gui/src/app/components/inspector/header/test-header.js [hunk 2]
```diff
@@ -63,42 +63,32 @@ describe('InspectorHeader', () => {
...
       <ul className="inspector-header__inline-list">
-        <li className='inspector-header__list-item'>
+        <li className="inspector-header__list-item">
           <a onClick={sinon.stub()}>
-            Get started
+            Charm details
           </a>
         </li>
-        <li className="inspector-header__list-item">
+        <li className='inspector-header__list-item'>
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/header/test-header.js]: double quotes

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**